### ISSUE TITLE
Enchantment with betterplayer

### ISSUE THREAD

**Issue by sudhanshugairola** [XuanTung95/simple_platform_view#1]
Can you please make this more powerfull with betterplayer because that use flutter_textureview which doesn't provide actual color even colour washout on HDR video.
That will help lots of users.
Developer failed to initialised platform view on Android side to provide a native view.

https://pub.dev/packages/extended_betterplayer

**Comment by XuanTung95** [comment]
Does these video_player plugin use texture to render instead of platformview?

**Comment by sudhanshugairola** [comment]
Yes both player doesn't use platformview

**Comment by sudhanshugairola** [comment]
Video_player not use much. But betterplayer is using widely by user they feel uncomfortable with flutter Texture. Platform view solve lot of issues like black or green screen sometime, HDR and Dolby vision color washout issue, and some time SDR video colour profiling issue.

You are expert and you have experience in platformview with better performance that why I approach to you. Hope you will work on better_player

**Comment by XuanTung95** [comment]
I think texture is good for video player plugin, the author did not use platform view so I cannot do anything. And I dont understand why using platform view is better than flutter texture.

**Comment by sudhanshugairola** [comment]
1. HDR and Dolby vision video color washout issue
2. Sometime Black and green screen issue
3. SDR video colour profiling mismatched

That all issue due to texture

**Comment by sudhanshugairola** [comment]
You can try on https://pub.dev/packages/extended_betterplayer

**Comment by XuanTung95** [comment]
I mean I dont know what is the root problem, it may come from exoplayer or some configuation. You can try to create a plugin using exoplayer platformview to test.

**Comment by sudhanshugairola** [comment]
Yes always tested flutter_playout that based on platformview with exoplayer
that work good

**Comment by XuanTung95** [comment]
If so you can modify better_player to use flutter platform view instead of texture and if it is not working you can try this plugin. Please note that this plugin has a position synchronization issue as mentioned in readme. Also, flutter has 3 platform view modes in Android, you should test with all of these modes.

I will maintain this plugin but I won't update better_player or other plugins. The author or contributor should do it.

**Comment by sudhanshugairola** [comment]
I don't know about native code bridging

**Comment by XuanTung95** [comment]
there is doc here https://docs.flutter.dev/platform-integration/android/platform-views
You could check PlayerViewFactory in flutter_playout

**Comment by sudhanshugairola** [comment]
Great
